SDK v2 migration guide.md

```diff
@@ -38,10 +38,13 @@ but users are still encouraged to migrate to version 2 when they have the chance
 | Version 1 API | Equivalent version 2 API |
 |:---|:---|
 | `DeviceClient` | `IotHubDeviceClient` |
-| `SetConnectionStatusChangesHandler` | `SetConnectionStatusChangeHandler` |
+| `DeviceClient.SendEventAsync(...)` | `IotHubDeviceClient.SendTelemetryAsync(...)` |
```

Comment:
On this note, should we also update (Incoming/Outgoing)Message to (Incoming/Outgoing)Telemetry?

Reply:
Incoming wouldn't be telemetry, but OutgoingMessage could be renamed to Telemetry. Like that?

Reply:
That makes sense. We do call it "SetIncomingMessageCallbackAsync".

Reply:
On second thought, I think TelemetryMessage instead of just Telemetry. Good?